runtime/moonbeam/tests/integration_test.rs

```diff
@@ -1220,6 +1223,46 @@ fn ethereum_invalid_transaction() {
 	});
 }
 
+#[test]
+fn initial_gas_fee_is_correct() {
+	use fp_evm::FeeCalculator;
+
+	ExtBuilder::default().build().execute_with(|| {
+		let multiplier = TransactionPayment::next_fee_multiplier();
+		assert_eq!(multiplier, Multiplier::from(1u128));
+
+		assert_eq!(
+			TransactionPaymentAsGasPrice::min_gas_price(),
+			(
+				125_000_000_000u128.into(),
+				Weight::from_ref_time(25_000_000u64)
+			)
+		);
+	});
+}
+
+#[test]
+fn min_gas_fee_is_correct() {
+	use fp_evm::FeeCalculator;
+	use frame_support::traits::Hooks;
+
+	ExtBuilder::default().build().execute_with(|| {
+		pallet_transaction_payment::NextFeeMultiplier::<Runtime>::put(Multiplier::from(0));
+		TransactionPayment::on_finalize(System::block_number()); // should trigger min to kick in
+
+		let multiplier = TransactionPayment::next_fee_multiplier();
+		assert_eq!(multiplier, Multiplier::from(1u128));
+
+		assert_eq!(
+			TransactionPaymentAsGasPrice::min_gas_price(),
+			(
+				125_000_000_000u128.into(),
+				Weight::from_ref_time(25_000_000u64)
+			)
+		);
```

Comment:
@tgmichel this should address your earlier comment. It is indeed `125 gwei`, but this should be in line with other runtimes when you take the `SUPPLY_FACTOR` into account.

Does `125 gwei` still seem unreasonable?